synapse/models/files.py

```diff
@@ -699,6 +702,42 @@ def getModelDefs(self):
                         'doc': 'The file offset to the beginning of the section'}),
                 )),
 
+                ('file:mime:lnk', {}, (
+                    ('flags', ('int', {}), {
+                        'doc': 'The flags specified by the LNK header that control the structure of the LNK file.'}),
+                    ('entry:primary', ('file:path', {}), {
+                        'doc': "The primary file path contained within the FileEntry structure of the LNK file."}),
+                    ('entry:secondary', ('file:path', {}), {
+                        'doc': "The secondary file path contained within the FileEntry structure of the LNK file."}),
```

Comment:
nit, but it looks like you've got some mixed use of `"` and `'` .  Tweak them to all be `'` :D